benchmark/daisy/daisy.jl

```diff
@@ -48,23 +48,24 @@ for name in keys(DAISY_ND)
     RELPREC[name] = Dict()
     dom = DAISY_ND[name]["dom"]
     ref = DAISY_ND[name]["ref"]
-    for m in [2, 5, 10]
+    for ord in [2, 5, 10]
         varnames, numvars = DAISY_ND[name]["vars"], DAISY_ND[name]["numvars"]
-        vars = set_variables(Float64, varnames, order=m, numvars=numvars)
+
+        vars = set_variables(Float64, varnames, order=ord, numvars=numvars)
         f = DAISY_ND[name]["f"]
         p = f(vars...)
 
-        RESULTS[name]["order $m"] = BenchmarkGroup()
-        RELPREC[name]["order $m"] = Dict()
+        RESULTS[name]["order $ord"] = BenchmarkGroup()
+        RELPREC[name]["order $ord"] = Dict()
 
-        RESULTS[name]["order $m"]["evaluate"] = @benchmarkable evaluate($p, $dom) setup=(
-            vars = set_variables(Float64, $varnames, order=$m, numvars=$numvars); p = $f(vars...))
+        RESULTS[name]["order $ord"]["evaluate"] = @benchmarkable evaluate($p, $dom) setup=(
```

Comment:
Same comment as above regarding `evaluate`: it should be computed as differences from the expansion point.